### PR TITLE
CodeIgniter Login

### PR DESCRIPTION
**Pull request by aaronpk**
This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [X] A title that's approved by Dev Marketing
- [X] A URL that's approved by Dev Marketing
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+
+The schema is simple - a `users` table (with an id and email), and a `news` table (with an `id`, `title`, `slug`, `text` of the article and a `user_id` for the article author). Note that there is no password field in the `users` table because user authentication will be handled by Okta.[a]
+
+## Okta Registration and Application Setup
+
+Before proceeding with the CodeIgniter application, take some time to set up your Okta account and application. Okta makes [identity management](https://developer.okta.com/product/user-management/) easier, more secure, and more scalable than you're used to. Okta's API service allows you to create, edit, and securely store user accounts and data, and connect them with one or more applications.
+
+[Register for a forever-free developer account](https://developer.okta.com/signup/), then log in to your Okta account, go to the Applications menu item and click the **Add Application** button:
+
+{% img blog/php-codeigniter-login/add-application.png alt:"Click the Add Application button" width:"300" align:"center" %}
```
Comment (mraible) [inline _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md]: This image is not center-aligned.

File: _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md [hunk 2]
```diff
@@ -0,0 +1,845 @@
...
+require 'vendor/autoload.php';
+use Dotenv\Dotenv;
+
+$dotenv = Dotenv::create(__DIR__);
+$dotenv->load();
+```
+
+Add the following line at the end of `public/index.php`, but before the default CodeIgniter bootstrapper: 
+
+```
```
Comment (mraible) [inline _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md]:
```suggestion
```php
```

File: _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md [hunk 3]
```diff
@@ -0,0 +1,843 @@
...
+    }
+```
+
+Note how the route is guarded by checking if there is a logged-in user before allowing the code to execute:
+
+```php
+        if (! $this->username) {
+            redirect('login');
+        }
+```
```
Comment (mraible) [inline _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md]: I would left-align this code block.

Comment (aaronpk) [reply _source/_posts/2019-10-28-simple-secure-authentication-with-codeigniter.md]: this indentation matches the code in the file so I like to leave it as is

**Pull request merged**